pkg/volume/cephfs/cephfs.go

```diff
@@ -397,12 +377,43 @@ func (cephfsVolume *cephfs) execFuseMount(mountpoint string) error {
 	}
 	src += hosts[i]
 
+	configArgs := []string{}
```

Comment:
Why are all these config options being set? Doesn't seem relevant to stated goal of the PR.